app-office/texstudio/texstudio-4.8.1.recipe

```diff
@@ -43,41 +43,42 @@ REQUIRES="
 	lib:libX11$secondaryArchSuffix
 	lib:libXext$secondaryArchSuffix
 	lib:libz$secondaryArchSuffix
-	# TeXlive packages
-	texlive_core$secondaryArchSuffix
-	texlive
-	texlive_basic
-	texlive_bibtexextra
-	texlive_binextra
-	texlive_fontsextra
-	texlive_fontsrecommended
-	texlive_formatsextra
-	texlive_games
-	texlive_humanities
-	texlive_latex
-	texlive_latexextra
-	texlive_latexrecommended
-	texlive_mathscience
-	texlive_music
-	texlive_pictures
+	# TeXlive packages, don't force this on the user while installing TeXstudio
+	# maybe create a small script to install the seperate packages?
+#	texlive_core$secondaryArchSuffix
+#	texlive
+#	texlive_basic
+#	texlive_latex
+#	texlive_bibtexextra
+#	texlive_binextra
+#	texlive_fontsextra
+#	texlive_fontsrecommended
+#	texlive_formatsextra
+#	texlive_games
+#	texlive_humanities
+#	texlive_latexrecommended
+#	texlive_latexextra
+#	texlive_pictures
+#	texlive_mathscience
+#	texlive_music
```

Comment:
@jmairboeck any thoughts on this, I noticed when installing without these packages around it's really hard to tell if HaikuDepot is still running or not. :)

Reply:
I think you are right. The texlive packages shouldn't be a hard dependency.

Reply:
Thanks for confirming. :)